### ISSUE TITLE
Install of optional-requirements.txt fails in Python 3.10

### ISSUE THREAD

**Issue by LiamConnors** [plotly/plotly.py#3484]
Following the steps in https://github.com/plotly/plotly.py/blob/doc-prod/contributing.md, I created a new Conda env and started installing the requirements. `requirements.txt` installed fine, but `optional-requirements.txt` failed.

`Failed to get options via gdal-config:` during installation of fiona>=1.8

The env had the following before install of `requirements.txt` and `optional-requirements.txt`:

```
setuptools-58.0.4
pip-21.2.4
tzdata-2021e
python-3.10.0
certifi-2020.6.20
ncurses-6.3
```
[Full error](https://github.com/plotly/plotly.py/files/7584777/error.txt)

**Comment by justinlblc** [comment]
Hello LiamConnors, my collaborators and I ran into the same issue. We found a solution.

When you create your environment, you have to specify a different version of Python, which is comptatible with Fiona. For instance, all versions between 3.9 (included) and 3.10 (not included) are compatible, which explains why it fails with Python 3.10.

```

conda create -n your-env-name python=3.9.7

```

I wish that this comment will help you.

Something else: if you run into an issue with the next step (pip install -r packages/python/plotly/requirements.txt), it may be because of the version of NodeJS. The version 17 isn't compatible, and you should use a previous one:

```

conda install -c conda-forge nodejs=16

```

**Comment by gvwilson** [comment]
Hi - we are trying to tidy up the stale issues and PRs in Plotly's public repositories so that we can focus on things that are still important to our community. Since this one has been sitting for several years, I'm going to close it; if it is still a concern, please add a comment letting us know what recent version of our software you've checked it with so that I can reopen it and add it to our backlog. Thanks for your help - @gvwilson
